.rustfmt.toml

```diff
@@ -0,0 +1,64 @@
+max_width = 90 # changed
+hard_tabs = false
+tab_spaces = 4
+newline_style = "Auto"
+use_small_heuristics = "Default"
+indent_style = "Block"
+wrap_comments = false
+format_code_in_doc_comments = false
+comment_width = 80
+normalize_comments = true # changed
+normalize_doc_attributes = false
+format_strings = false
+format_macro_matchers = false
+format_macro_bodies = true
+empty_item_single_line = true
+struct_lit_single_line = true
+fn_single_line = false
+where_single_line = false
+imports_indent = "Block"
+imports_layout = "Vertical" # changed
+imports_granularity = "Crate" # changed
```

Comment:
Example from our codebase("Crate"):
<img width="399" alt="изображение" src="https://user-images.githubusercontent.com/18346821/186181331-ccf0f6c5-49a6-4c40-8c03-14ec44dee161.png">

Example from our codebase("Module"):
<img width="673" alt="изображение" src="https://user-images.githubusercontent.com/18346821/186181606-998d30a0-eb6a-4b23-b393-2398cb862ee2.png">


I'm okay with both variants but prefer more "Crate" because I can easy collapse imports from one crate or add a `cfg` flag for whole crate=)

Reply:
the cfg flag can go both ways. For example, it is also likely that a cfg flag will affect a subset of imports within a crate, rather than the entire crate. 


```
use crate::{ThingA, ThingB}
#[cfg(feature = "feature-c")]
use crate::ThingC
```

Will rustfmt avoid collapsing the import if we have a cfg flag on a specific set of imports from a crate?

Reply:
Yes, it will not touch `cfg` imports. Also, if you have an "enter" between to import section, it will not merge them into one, but it will collapse and sort all inside each section(the same for `cfg`).